src/main/java/Angela/Angela.java

```diff
@@ -0,0 +1,190 @@
+                        @@( /////////////////////////&&@&&&&&&&&&&@@@@@@@@@@@%/@                  @@@  \s
+                        @@ #////////////////////////@@&@&&&&&&&&&@@@           @@                  @@  \s
+                        @@ /////////////////////@@@ @@@@&&&&&&&&&@              @@@ %              @@@ \s
+                        @@ //////////////////@@       @@&&&&&&&&&@                @@.              ,@@ \s
+                        @@,///////////////@@@         @@&&&&&&&&@@                  @@ .            @@ \s
+                        @@%////////////@@             @@&&&&&&&&@                   @@@ *           &@@\s
+                        @@@//////////@@@              @@&&&&&&&@@                    *@@             @@\s
+                          @////////%@                 @@&&&&&&&@@                      @#           *@ \s
+                          @@//////@@                  @@&&&&&&@@                        @.         @@@ \s
+                          *@////@@@                   @@&&&&&&@@                        @@        @%*  \s
+                           @@//@@                     @@&&&&&&@@                         @@ *   @@#/   \s
+                           @@&@@                      @@&&&&&&@@                         @@@ #@@@      \s
+                            @@@                       @@&&&&&&@@                           @@@@@       \s
+                                                    @@&&&&&&&&@@                           @@          \s
+                                                  @@&&&&&&&&&&@@                                       \s
+                                                 @@@@#@&&&&&&&@@                                       \s
+                                                 @####&&&&&&&&@@                                       \s
+                                                  @@@@&&&&&&&&@@                                       \s
+                                                    @@&&&&&&&&%@                                       \s
+                                                    @##@&&&&@##@                                       \s
+                                                    @@########@@                                       \s
+                                                      @@####@@@                                        \s
+                        
+                        
+                Welcome, Manager. I am your assistant AI, Angela.
+                I am here to support you as it's your first day. 
+                I will provide practical advice and emotional support until you get used to your work.
+                
+                What is your request?                        
+                """
+        ), 7000);
+    }
+
+    //@@author Oleg Mikhailov-reused
+    //Reused from https://stackoverflow.com/questions/26311470/what-is-the-equivalent-of-javascript-settimeout-in-java
+    public static void setTimeout(Runnable runnable, int delay){
```

Comment:
```suggestion
    public static void setTimeout(Runnable runnable, int delay) {
```
To be honest I'm not sure about the proper procedures for editing reused code, but if possible then a space would be a good idea here.